modules/pages/scripts/templates.js

```diff
@@ -0,0 +1,15 @@
+(function () {
+    'use strict';
+
+    var module = angular.module('feincms-pages.templates', []);
```

Comment:
I think now is a good opportunity to make html templates and use `ngtemplates` to compile to this module name